ffi/core.cpp

```diff
@@ -21,10 +21,23 @@ API_EXPORT(void)
 LLVMPY_DisposeString(const char *msg) { free(const_cast<char *>(msg)); }
 
 API_EXPORT(LLVMContextRef)
-LLVMPY_GetGlobalContext() { return LLVMGetGlobalContext(); }
+LLVMPY_GetGlobalContext() {
+  auto context = LLVMGetGlobalContext();
+#if LLVM_VERSION_MAJOR > 14
+  LLVMContextSetOpaquePointers(context, false);
```

Comment:
Note to other reviewers - this does result in opaque pointers being disabled for the global context multiple times, but this operation is idempotent.